content/SCALE/SCALEUIReference/Credentials/LocalUsersScreensSCALE.md

```diff
@@ -90,7 +90,7 @@ Built-in users (except the **root** user) do not include the **Home Directory Pe
 {{< truetable >}}
 | Setting | Description |
 |---------|-------------|
-| **Home Directory** | Enter or browse to enter the path to the home directory for this user. If the directory exists and matches the **Username**, it is set as the home directory for the user. When the path does not end with a subdirectory matching the username, a new subdirectory is created if **Create Home Directory** is selected (enabled). The full path to the user home directory shows in this field on the **Edit User** screen for this user. The default location is **/var/empty**. |   
+| **Home Directory** | Enter or browse to enter the path to the home directory for this user. If the directory exists and matches the **Username**, it is set as the home directory for the user. When the path does not end with a subdirectory matching the username, a new subdirectory is created if the **Create Home Directory** option above the **Save** button is selected. The full path to the user home directory shows in this field on the **Edit User** screen for this user. The default location is **/var/empty**. |   
```

Comment:
This an example of using the UI element as the noun incorrectly. Legacy text has a lot of these instances. 
```suggestion
| **Home Directory** | Enter or browse to enter the path to the home directory for this user. If the directory exists and matches the value in **Username**, it is set as the home directory for the user. When the path does not end with a subdirectory matching the username, a new subdirectory is created if the **Create Home Directory** option above the **Save** button is selected. The full path to the user home directory shows in this field on the **Edit User** screen for this user. The default location is **/var/empty**. |   
```